torch/csrc/api/include/torch/data/datasets/chunk.h

```diff
@@ -0,0 +1,439 @@
+#pragma once
+
+#include <torch/data/datasets/stateful.h>
+
```

Comment:
I don't see that you resolved this

Reply:
I am preparing a PR to remove the LockedSampler that we all did not like. I will include this in that too.